x-pack/test_utils/testbed/testbed.ts

```diff
@@ -138,33 +139,23 @@ export const registerTestBed = <T extends string = string>(
         });
       };
 
-      const waitFor: TestBed<T>['waitFor'] = async (testSubject: T, count = 1) => {
+      const waitForFn: TestBed<T>['waitForFn'] = async (predicate, errMessage) => {
         const triggeredAt = Date.now();
 
-        /**
-         * The way jest run tests in parallel + the not deterministic DOM update from React "hooks"
```

Comment:
Why do we want to remove this comment?

Reply:
This is a mistake, but my current finding is that we should stop using this hack. After the form lib tests are done I will go back to this. `setTimeout` are not good with jest unit testing and CI.